cmd/locate.go

```diff
@@ -2,54 +2,83 @@ package cmd
 
 import (
 	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	"golang.org/x/mod/modfile"
+	"golang.org/x/mod/module"
 
 	adlr "github.com/blocky/adlr/pkg"
 	"github.com/blocky/adlr/pkg/gotool"
-	"github.com/spf13/cobra"
 )
 
 var LocatedFile string
+var ProjectDir string
 var ExemptMods []string
 
 var locateCmd = &cobra.Command{
 	Use:   "locate",
 	Short: "Locate dependency licenses",
 	Long:  "Outputs a json file containing located licenses.",
 	Run: func(cmd *cobra.Command, args []string) {
-		err := Locate(BuildlistFile, LocatedFile)
+		err := Locate(LocatedFile, ProjectDir)
 		ExitOnErr(err)
 	},
 }
 
 func init() {
-	locateCmd.Flags().StringVarP(
-		&BuildlistFile, "buildlist", "b", "./buildlist.json",
-		"Path of module build list in json format",
-	)
 	locateCmd.Flags().StringVarP(
 		&LocatedFile, "located", "l", "./located-licenses.json",
 		"Output file containing located licenses",
 	)
+	locateCmd.Flags().StringVarP(
+		&ProjectDir, "project-dir", "p", "./",
+		"Path to the Go project whose dependencies you wish to audit",
+	)
```

Comment:
Instead of a buildlist made by `go list` adlr uses the `go.mod` and `vendor` folder.